clients/python-wrapper/tests/integration/test_sanity.py

```diff
@@ -58,12 +60,69 @@ def test_branch_sanity(storage_namespace, setup_repo):
     with obj.reader() as fd:
         assert fd.read() == initial_content
 
-    new_branch.delete()
+    obj.upload(override_content)
```

Comment:
Feels like this goes beyond the scope of sanity - can you add it to the test_object module?

Reply:
@N-o-Z test_object? but it's used to test branch capabilities

Reply:
test_branch...